.github/workflows/update-be.yml

```
@@ -0,0 +1,36 @@
+name: Update BE QA version
+on:
+  push:
+    tags:
+    - v[0-9]+.[0-9]+.[0-9]+
```

```suggestion
    - @stream-io/video-client@v[0-9]+.[0-9]+.[0-9]+
```

I think we should be more specific here, as we don't want to open PRs for non-client related releases, right?